app/admin/reports.rb

```diff
@@ -34,6 +35,26 @@
 
   controller do
     
+    def complete_apps_demographic_report
+      query = "SELECT ad.country,
+      (CASE WHEN ad.gender = '' THEN NULL ELSE 
+      (SELECT genders.name FROM genders WHERE CAST(ad.gender AS UNSIGNED) = genders.id) END) as gender,
+      e.year_in_school,
+      (CASE WHEN ad.demographic = '' THEN NULL ELSE 
+      (SELECT demographics.name FROM demographics WHERE CAST(ad.demographic AS UNSIGNED) = demographics.id) END) AS demographic,
+      e.international
+      FROM enrollments AS e 
+      LEFT JOIN users AS u ON e.user_id = u.id
+      LEFT JOIN applicant_details AS ad ON ad.user_id = e.user_id
+      WHERE e.application_status = 'application complete' AND e.campyear = #{CampConfiguration.active.last.camp_year} ORDER BY country, gender, year_in_school"
+      title = "demographic"
```

Comment:
Change the title to be different from the other demographic report
I would suggest 
"complete_apps_demographic_report"
-and on line 70 
"enrolled_student_demographic_report"

Reply:
Thanks for noticing - edited.